internal/api/schemas/v1.yaml

```diff
@@ -0,0 +1,47 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Consul API Gateway
+  description: Consul API Gateway Management API
+  contact:
+    name: Consul API Gateway Team
+    email: consul-api-gateway@hashicorp.com
+    url: http://hashicorp.com
+  license:
+    name: Mozilla Public License 2.0
+    url: https://www.mozilla.org/en-US/MPL/2.0/
```

Comment:
I see that nomad-openapi is linking to the license in the repo. Wondering if we should do the same

https://github.com/hashicorp/nomad-openapi/blob/ccfbbe2cdd4a2e4d65db0204f9d3a5b51215d2b3/v1/openapi.yaml#L5658

Reply:
I think it's effectively saying the same thing since the LICENSE.md is just a copy of MPL 2, but went ahead and updated to point to our repo.